region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -1246,18 +1975,91 @@
         {"heatFrames": 60}
       ]
     },
+    {
+      "link": [4, 1],
+      "name": "Start Shinecharged, Leave Shinecharged",
+      "startsWithShineCharge": true,
+      "requires": [
+        "canShinechargeMovementTricky",
```

Comment:
Why tricky?

Reply:
Not sure, I think at one point I had in mind that carrying the shinecharge through the item collection should stay Expert like it was with the notable. I changed my mind but then probably forgot I put these in. Downgraded them to `canShinechargeMovementComplex` now.